Sources/swiftpm-xctest-helper/main.swift

```diff
@@ -0,0 +1,108 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
```

Comment:
It would be a good idea to have a comment here explaining what this tool is for. What I would probably do is factor out a "main" function, put it at the top, and just make the documentation comment for that explain what it is doing.